docs/howto-inhibit-upgrade.md

```diff
@@ -1,6 +1,6 @@
 # Upgrade Inhibition
 ### Process Inhibition
-With latest changes on Leapp and with new actors added to main Offline Repository, any actor can inhibit the upgrade process by producing a message with a high enough severity in case a problem is found. The message model in question is named [CheckResult](https://github.com/oamg/leapp-repository/blob/master/repos/offline-upgrade/models/checkresult.py) and can have the following severities: **Info**, **Warning** and **Error**. If a message with a severity equal to **Error** is generated it will stop the process showing a message to the user. All [CheckResult](https://github.com/oamg/leapp-repository/blob/master/repos/offline-upgrade/models/checkresult.py) messages will be part of a report that will be available to the user after process end.
+With latest changes on Leapp and with new actors added to the el7toel8 Leapp repository, any actor can inhibit the upgrade process by producing a specific message when a problem is found. The message model to use in this case is [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py). The Inhibitor messages are collected in the Report phase, so if there's at least one Inhibitor message produced before this phase, the upgrade will be stopped in the Reports phase. It means that any Inhibitor message produced after the Report phase will have inhibiting effect. The details mentioned in the Inhibitor messages will be part of the report available to the user to review.
```

Comment:
Inhibitor messages after Report phase will NOT have inhibiting effect, right?

Reply:
@jmikovic  Correct

Reply:
Right, that's what I wanted to write there :) Thanks.